src/main/java/com/puppycrawl/tools/checkstyle/gui/TreeTable.java

```diff
@@ -108,7 +108,7 @@ public TreeTable(ParseTreeTableModel treeTableModel) {
         // the table.
         if (tree.getRowHeight() < 1) {
             // Metal looks better like this.
-            setRowHeight(getRowHeight());
+            tree.setRowHeight(getRowHeight());
```

Comment:
how can we prove that it will work after fix?
I would suggest to just extract variable and set it, like
```
int height = getRowHeight();
setRowHeight(height);
```
just to be on a safe side

Reply:
You have a point there, that's a nice workaround.
Changes are pushed. Coverage went down from `0.90` to `0.89` but that is not an issue here I think.

Reply:
I couldn't reproduce the case for `tree.getRowHeight() < 1`. This may be just a dead code.

Reply:
Me neither. Both `tree` and `this`'s `rowHeight`s are set to 16 initally. I suppose this if-statement is there in case some of the set methods above it mess something up?

I ran `git blame` on the lines in question but the if-statement was written all the way back in 2002 along with other files so no information about it. The commit is not referencing an issue or anything.